spec/controllers/course_summaries_controller_spec.rb

```diff
@@ -54,22 +54,31 @@
       before :each do
         3.times { create(:assignment_with_criteria_and_results) }
         2.times { create(:grade_entry_form_with_data) }
+
+        # Ensure statistics exist for one assignment
+        assignments = Assignment.all
+        assignments.first.update_results_stats
```

Comment:
Hm. It looks like this should be call on all assignments (since they all have results), so the best approach would be to add this to the factory itself in an `after_create` callback.

Reply:
@ignaspa did you not move this?

Reply:
I did! I think I missed removing it in that instance, sorry.